src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    private static final int ARRAY_SIZE = 6;
+    private static final double MAX_SIZE = 10.0;
+    private static final Random RANDOM = new Random();
+    private static double size = RANDOM.nextDouble() * MAX_SIZE + 1;
```

Comment:
your size is generated randomly only once. Let's generate new random size each time getRandomFigure method is called